pages/events/search.js

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from "next/router";
+import EventList from "@/components/EventList";
+import Button from "@/components/Button";
+export default function Search() {
+  const router = useRouter();
+  const { q } = router.query;
+
+  return (
+    <>
+      <Button onClick={() => router.back()}>Back</Button>
+      <h1>Your Search for &apos;{q}&apos; revealed the following results:</h1>
```

Comment:
Please check [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de) again and keep in mind we are working on SPAs. H1 shall only appear once in your application.